code/modules/projectiles/guns/projectile.dm

```diff
@@ -313,7 +313,8 @@
 //		if(in_chamber && loaded.len)
 //			to_chat(usr, "It also has a chambered round." {R})
 	if(istype(silenced, /obj/item/gun_part/silencer))
-		to_chat(user, "<span class='warning'>It has a suppressor attached to the barrel.</span>")
+		var/obj/item/gun_part/silencer/A = silenced
+		to_chat(user, "<span class='warning'>It has a [A.name] attached to the barrel.</span>")
```

Comment:
I'm not familiar with text macros but I think it should be
```suggestion
		to_chat(user, "<span class='warning'>It has \a [A.name] attached to the barrel.</span>")
```

Reply:
It should be `\a [A]`